src/EPR.PRN.Backend.API/Handlers/UpdateRegulatorApplicationTaskHandler.cs

```diff
@@ -0,0 +1,15 @@
+using EPR.PRN.Backend.API.Commands;
+using EPR.PRN.Backend.Data.DataModels.Registrations;
+using EPR.PRN.Backend.Data.Interfaces.Regulator;
+
+namespace EPR.PRN.Backend.API.Handlers;
+
+public class UpdateRegulatorApplicationTaskHandler : UpdateRegulatorTaskHandlerBase<UpdateRegulatorApplicationTaskCommand, IRegulatorApplicationTaskStatusRepository, RegulatorApplicationTaskStatus>
+{
+    public UpdateRegulatorApplicationTaskHandler(IRegulatorApplicationTaskStatusRepository repository)
+        : base(repository)
+    {
+    }
+}
+
+
```

Comment:
Could use a primary constructor here perhaps?

```suggestion
public class UpdateRegulatorApplicationTaskHandler(IRegulatorApplicationTaskStatusRepository repository)
    : UpdateRegulatorTaskHandlerBase<UpdateRegulatorApplicationTaskCommand, IRegulatorApplicationTaskStatusRepository,
        RegulatorApplicationTaskStatus>(repository);
```